de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/console/InviteCommand.java

```diff
@@ -0,0 +1,46 @@
+package de.fu_berlin.inf.dpp.server.console;
+
+import de.fu_berlin.inf.dpp.net.xmpp.JID;
+import de.fu_berlin.inf.dpp.session.ISarosSessionManager;
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.log4j.Logger;
+
+public class InviteCommand extends ConsoleCommand {
+  private static final Logger LOG = Logger.getLogger(InviteCommand.class);
+  private final ISarosSessionManager sessionManager;
+
+  public InviteCommand(ISarosSessionManager sessionManager) {
+    this.sessionManager = sessionManager;
+  }
+
+  @Override
+  public String identifier() {
+    return "invite";
+  }
+
+  @Override
+  public String help() {
+    return "invite <JID>... - Invite users to session";
+  }
+
+  @Override
+  public void execute(String command, PrintStream out) {
+    try {
+      List<JID> jids =
+          Arrays.asList(command.split(" "))
+              .stream()
+              .skip(1)
+              .map(x -> new JID(x))
```

Comment:
Check if JID is valid with `XMPPUtils.validateJID(JID)`?
You could either
- just filter out illegal JIDs before the collect or
- check after the collect and then abort the invitation process and print an error message if invalid JIDs are found.

I am not sure which option would be preferable.

Reply:
I decided to filter them out. This console is mostly for testing anyway and the logs state which users got invited.

Reply:
But the information on who was invited is pretty dull. The important information is: who was not invited. I know it is intended for 'testing purposes', but could you please add some output information about who was not invited, because of invalid JID?

Reply:
Added